README.md

```diff
@@ -102,13 +81,6 @@ High level `make` targets:
 * `system-test`: run the networking/"sanity" tests. Specify
   `CONTIV_NODE_OS=centos` to test on centos instead of ubuntu.
 
-#### Resource Allocation
```

Comment:
Maybe we should have a CONTRIBUTING.md?

On 16 Sep 2015, at 10:47, Vipin Jain wrote:

>> @@ -102,13 +81,6 @@ High level `make` targets:
>> * `system-test`: run the networking/"sanity" tests. Specify
>> `CONTIV_NODE_OS=centos` to test on centos instead of ubuntu.
>>
>> -#### Resource Allocation
>
>> Just wondering, shouldn't we continue to keep this as we still allow 
>> this, right?
>
> Yes we do that, however I am thinking we keep the front page to be 
> usage centric for someone to get going and explain the details 
> separately.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/netplugin/pull/147/files#r39661947